pennylane/devices/default_qubit_tf.py

```diff
@@ -159,6 +159,11 @@ def __init__(self, wires, *, shots=1000, analytic=True):
         # prevent using special apply method for this gate due to slowdown in TF implementation
         del self._apply_ops["CZ"]
 
+        # prevent using special apply methods when using more than 8 wires due to limitations
+        # with TF slicing
+        if wires > 8:
+            self._apply_ops = {}

```

Comment:
Would we want the unaffected (`PauliX` and `SWAP`) to remain in there?

Reply:
Potentially, but I think the gain is relatively small since they are less common gates, so maybe better to turn off for clarity.